image-loader/app/model/Uploader.scala

```diff
@@ -329,7 +329,7 @@ class Uploader(val store: ImageLoaderStore,
       updateMessage = UpdateMessage(subject = "image", image = Some(imageUpload.image))
       _ <- Future { notifications.publish(updateMessage) }
       // TODO: centralise where all these URLs are constructed
-      uri = s"${config.apiUri}/images/${uploadRequest.imageId}"
+      uri = s"${config.rootUri}/follow/${uploadRequest.imageId}"
```

Comment:
Following the comment below, this should be the `GET /uploadStatus/:imageId ` endpoint.

Reply:
Would we also need to update the [`quarantineFile` endpoint](https://github.com/guardian/grid/blob/9f288a37bf26876b0103692c5ca115ad4f019e99/image-loader/app/model/QuarantineUploader.scala#L40-L51) to make it make it match this. It might be better to push this change up [to here](https://github.com/guardian/grid/blob/552671ef1445f02dbfd2aa657d99ed4cae8678b8/image-loader/app/controllers/ImageLoaderController.scala#L55-L57)?